### DISCUSSION TITLE
Please help! error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8) error: 7470 bytes of body are still expected

### DISCUSSION THREAD

**Discussion by JaneIva** [Homebrew/discussions#5440] [Getting started]
Guys, please help! I don't understand anything about it.
These are the errors indicated by the Terminal after paste the copied link from Homebrew

`Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R jane:admin /opt/homebrew
==> Downloading and installing Homebrew...
remote: Enumerating objects: 271043, done.
remote: Counting objects: 100% (734/734), done.
remote: Compressing objects: 100% (424/424), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 7470 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packets
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
Failed during: /usr/bin/git fetch --force origin`

Mac OS ( Sonoma 14.5 ) M1

Please help me install, thanks in advance

**Comment by krehel** [comment]
Please do not comment on an existing issue with the same name and then open a new issue for the equivalent, seeing as this same comment is [here](https://github.com/orgs/Homebrew/discussions/4069#discussioncomment-9759903)
